packages/react-jss/src/createHoc.js

```diff
@@ -84,39 +83,36 @@ export default function createHOC<
   const noTheme = {}
   const managerId = managersCounter++
   const manager = new SheetsManager()
+  const ThemeConsumer = (theming && theming.context.Consumer) || ThemeContext.Consumer
 
   // $FlowFixMe: DefaultProps is missing in type definitions
   const {classes: defaultClasses = {}, ...defaultProps} = {...InnerComponent.defaultProps}
 
   class Jss extends Component<OuterPropsType, State> {
     static displayName = `Jss(${displayName})`
 
-    static InnerComponent = InnerComponent
-
-    static contextTypes = contextTypes
-
     static propTypes = {
       innerRef: PropTypes.func
     }
 
-    static defaultProps = {
-      ...defaultProps,
-      theme: noTheme
+    static defaultProps = defaultProps
+
+    static getTheme(props) {
```

Comment:
do you need this one to be static property of the component? or can this be  just a function scoped to the module

Reply:
No, it could be moved outside

Reply:
I would do it to avoid making it public